src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -127,7 +129,7 @@ public FormValidation doCheckCredentialsId(@AncestorInPath Item project,
             return FormValidation.ok();
         }
 
-        public FormValidation doCheckUrl(@AncestorInPath Item project,
+        public FormValidation doCheckUrl(@AncestorInPath Job project,
```

Comment:
IIRC we had to move this to `Item` so that the pipeline snipit generator would work from Folders (which are not a Job)